docs/dev/backend-setup.md

```
@@ -0,0 +1,140 @@
+> **Note:** The backend uses an in-memory store — all data is lost when the container stops.
+
+### 2. Provision test users
+
+The backend comes with a provisioning script that creates test accounts (`alice`, `bob`, `brian`, `charlotte`, `david`, `emma`) all at `example.com` with their name as the password:
+
+```bash
+docker exec tmail-backend bash /root/conf/integration_test/provisioning.sh
+```
+
+Sample credentials:
+| Email | Password |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)